editorconfig_plugin/shared.py

```diff
@@ -1,8 +1,10 @@
-from os.path import abspath
+import os
 import sys
 import logging
 
-editorconfig_path = abspath('editorconfig-core-py/')
+plugin_dir = os.path.dirname(__file__)
+lib_dir = os.path.join(plugin_dir, '..', 'editorconfig-core-py')
+editorconfig_path = os.path.realpath(lib_dir)
```

Comment:
Can you explain why you changed `abspath` to `realpath`?

Reply:
TBH probably because the SO answer I copied used `realpath()` :laughing: 

I think having `realpath()` dereference symbolic links feels "safer" to me, but in this case it probably makes no difference (either Python can load the library or it can't, whether or not the path has symbolic links). I can change it to use `abspath()` if you prefer.

Reply:
Thanks, please do so. I think it's just better to keep changes minimal.